flink-table/flink-sql-gateway-api/src/main/java/org/apache/flink/table/gateway/api/results/ExceptionInfo.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.gateway.api.results;
+
+import org.apache.flink.util.Preconditions;
+
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonCreator;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonProperty;
+
+/** ExceptionInfo for SQL gateway service. */
+public class ExceptionInfo {
+
+    private static final String FIELD_EXCEPTION_ROOT_CLASS_NAME = "root";
+    private static final String FIELD_EXCEPTION_MESSAGE = "message";
```

Comment:
This is not specified in FLIP.

Reply:
Fixed.
The ExceptionInfo contains the same attributes as the FLIP.

"exception": {

      "rootCause": "....",

     "exceptionStack": "..." 

}